src/species/corn.ts

```diff
@@ -54,23 +54,36 @@ const getLifestage = (agent: Agent): number => {
   return stage;
 };
 
-const cornInfectedTrait = new Trait({
-  name: 'infected',
-  possibleValues: [ true, false ],
-  default: false,
+const healthTrait = new Trait({
+  name: 'health',
+  min: 0,
+  max: 100,
```

Comment:
If we only update the default in the configuration, is the max value updated also?

Reply:
Good question -- probably not. I should check to see where/how the min/max are applied. It might make sense to synchronize min/max with any changes to the default.

Reply:
It looks like it wouldn't matter. The min/max values seem to be used when mutating and when generating random values, which I don't think we're using for any of our traits.